sdk/loadtestservice/azure-developer-loadtesting/README.md

```diff
@@ -144,43 +151,48 @@ except HttpResponseError as e:
 
 ### Uploading .jmx file to a Test
 ```python
-from azure.developer.loadtesting import LoadTestingClient
+from azure.developer.loadtesting import LoadTestAdministrationClient
 from azure.identity import DefaultAzureCredential
 from azure.core.exceptions import HttpResponseError
 
 TEST_ID = "some-test-id"  
-FILE_ID = "some-file-id"  
+FILE_NAME = "some-file-name.jmx"  
 
-client = LoadTestingClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+client = LoadTestAdministrationClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
 
 try:
 
-    result = client.load_test_administration.upload_test_file(TEST_ID, FILE_ID, open("sample.jmx", "rb"))
+    result = client.load_test_administration.upload_test_file(TEST_ID, FILE_NAME, open("sample.jmx", "rb"))
     print(result)
 except HttpResponseError as e:
     print("Failed with error: {}".format(e.response.json()))
 ```
 
 ### Running a Test
 ```python
-from azure.developer.loadtesting import LoadTestingClient
+from azure.developer.loadtesting import LoadTestRunClient
 from azure.identity import DefaultAzureCredential
 from azure.core.exceptions import HttpResponseError
 
 TEST_ID = "some-test-id"  
 TEST_RUN_ID = "some-testrun-id" 
 DISPLAY_NAME = "my-load-test-run"  
 
-client = LoadTestingClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+client = LoadTestRunClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
 
 try:
-    result = client.load_test_runs.create_or_update_test(
-        TEST_RUN_ID,
-        {
-            "testId": TEST_ID,
-            "displayName": DISPLAY_NAME,
-        },
+    testRunPoller = client.begin_create_or_update_test_run(
+    TEST_RUN_ID,
+    {
+        "testId": TEST_ID,
+        "displayName": "My New Load Test Run",
+    },
+    poll_for_test_run_status=True
     )
+
+    #waiting for test run status to be completed with timeout = 3600 seconds
+    result = testRunPoller.result(3600)
```

Comment:
Q: what does the result looked like if the poller times out at 3600 seconds

Reply:
@prativen `LROPoller.result()` works in a way that it waits for a maximum of given number of seconds, and then returns the last response received. So, if poller time's out at 3600 seconds then it would return the result of last request before the timeout.